ekip/nationalparks/management/commands/passes.py

```diff
@@ -73,17 +73,20 @@ def process_site(site, next_version):
         fs.state = site['state']
         fs.website = site['website']
         fs.annual_pass = annual
-        fs.senior_pass = senior
+        fs.senior_pass = annual
```

Comment:
This is confusing from a self-documenting code standpoint. Why is annual set to senior_pass? I'd either rename things, or add some comments to make this clear.